packages/react-core/src/next/components/Wizard/WizardToggle.tsx

```diff
@@ -28,11 +28,11 @@ export interface WizardToggleProps {
   activeStep: WizardControlStep;
   /** The WizardFooter */
   footer: React.ReactElement;
-  /** Custom WizardNav or callback used to create a default WizardNav */
-  nav: DefaultWizardNavProps | CustomWizardNavFunction;
   /** Navigate using the step index */
   goToStepByIndex: (index: number) => void;
-  /** The button's aria-label */
+  /** Custom WizardNav or callback used to create a default WizardNav */
+  nav?: DefaultWizardNavProps | CustomWizardNavFunction;
```

Comment:
Would it be possible/wise to have the nav and the footer both use the same pattern? Right now, the custom footer is a props object or a react node and the nav is a props object or a function.

Reply:
I don't think it's necessary, but we could make a `CustomWizardFooterFunction` if you want that returns `onNext`, `onBack`, and `onClose`. Thoughts on that change?

Reply:
That'd be helpful if people want a custom footer with custom buttons, right? so maybe that'd be wise

Reply:
I agree. That will make things complicated with how the footer is managed in context. We'd end up initializing footer's state variable as a function, which I'm not sure is even possible. Maybe I can try to do this in the next PR here where I'm updating a bunch of types (and including a new one called `CustomWizardNavItemFunction`):
https://github.com/patternfly/patternfly-react/pull/7915/files#diff-087f56845137e426d26fe8b189b155902a59de75ee7e2beff5c07db4d4f9c25f

Unless you think this is a blocking change for this PR.

Reply:
I don't think it's blocking since all this is beta :)

Reply:
Ok cool. I will address it in that other PR then, thanks!